releasenotes/notes/statevector-enable-iter-4652d7ce87f4d459.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
@jakelishman, if you could, please, have a look at this when you get a chance, as I'm not 100% sure if it's been done properly.
I'm comfortable both with Markdown and reStructuredText, and I've checked the rendering of the file itself and it is okay. However, I'm not sure if it plays nicely with Sphinx, as the latter only spat out an unhelpful 
```
build finished with problems, 4 warnings.
ERROR: InvocationError for command sphinx-build -W -T --keep-going -b html docs/ docs/_build/html (exited with code 1)
```
at the end of `reno report`, followed by `tox -edocs` (which took quite a while). The `release_notes.html` file has been updated, according to its timestamp, but I do not see the note corresponding to this PR in it.

Reply:
Reno associates notes with commit objects, so if you just had the note in the `git` staging index, it probably won't have been collected.  Don't worry - our docs CI build catches these sorts of errors.

I pushed a commit that simplified the release note. Sorry about that - I should have mentioned when I first asked that we usually try (try!) to keep them fairly short.  Your reST formatting looked fine to me - I don't know what exactly Sphinx would have been complaining about.

Reply:
@jakelishman, thanks for looking into this bit as well!

Could you please suggest how I could've performed the check for whether the reStructuredText and YAML formats are correct, and whether the resulting release note looks as one would expect it to look? Is there a faster way, in comparison to building the entire set of release notes for a single version?
I'm familiar with how to check for reST and YAML syntax (and respective rendering of the former, in isolation), but not sure how to check how those have been ultimately rendered as a piece of documentation, taking into account the codebase context etc.

Reply:
I can't remember exactly what reno looks for to include the note.  I think as long as the note is in the git staging index (i.e. it's been `git add`-ed), `reno` will pick it up.  You don't need to run `reno report` yourself - Sphinx will do that if you run `tox -edocs`, though if you update release notes, sometimes you need to force it to rebuild that page with a `touch docs/release_notes.rst` before.  `tox -edocs` should be much faster on subsequent runs, because Sphinx caches outputs that haven't changed.

To properly answer, it's something like:
- `git add` the release note
- `touch docs/release_notes.rst`
- `tox -edocs`

But honestly, if you're familiar with reST, I usually don't actually bother trying to build the documentation myself, and just let the CI check catch it if it wants to complain.